docs/scenarios/javaEE_to_quarkus/javaEE_to_quarkus.md

```diff
@@ -0,0 +1,329 @@
+application](https://github.com/konveyor-ecosystem/coolstore.git) to Quarkus, a
+task that involves not only technical translation but also considerations for
+deployment to Kubernetes. By the end of this demo, you will understand how
+Konveyor AI (Kai) can assist and expedite the modernization process.
+
+## Prerequisites
+
+- [VSCode](https://code.visualstudio.com/download)
+- [Git](https://git-scm.com/downloads)
+- [Kubernetes cluster (minikube)](https://minikube.sigs.k8s.io/docs/start/)
+- AI credentials
+- [Maven](https://maven.apache.org/install.html)
+- Java 21
+
+Additionally, you will need to have the Kai IDE plugin installed in VSCode.
+Download the latest from
+[here](https://github.com/konveyor/editor-extensions/releases).
+
+## Step 1: Setup
+
+### Installation
+
+Follow the steps in the [installation guide](../../installation.md) to install Kai. It will help you find the latest build and complete the setup.
+
+### Get a Demo App
+
+1. Clone the Cool-store application:
+
+   ```bash
+   git clone https://github.com/konveyor-ecosystem/coolstore.git
+   ```
+
+   Next, switch to the branch of the Coolstore app that's been partially migrated:
+
+   ```sh
+   git checkout partial-migration
+   ```
+
+2. Navigate to File > Open in VSCode and locate the folder we just cloned.
+   Make sure you have GenAI credentials before you start configuring Kai.
+
+### Configure Konveyor and Start the server
+
+Follow the steps in the [Configuration Guide](../../configuration.md) to set up Kai and start the server. The guide provides in-depth details on customizing the configuration to suit your project’s specific needs.
+
+> [!NOTE]
+> If you are following the guided demo for the Coolstore application, ensure you select the following targets during project configuration.
+
+We will analyze the Coolstore application using the following migration targets to identify potential areas for improvement:
+
+- cloud-readiness
+- jakarta-ee
+- jakarta-ee8
+- jakarta-ee9
+- quarkus
+
+To verify your target configuration, go to your project directory and open `/.vscode/settings.json`. This serves as a reference for how `settings.json` should look.
+
+```json
+{
+  "konveyor.analysis.labelSelector": "(konveyor.io/target=cloud-readiness || konveyor.io/target=jakarta-ee || konveyor.io/target=jakarta-ee8 || konveyor.io/target=jakarta-ee9 || konveyor.io/target=quarkus) || (discovery)"
+}
+```
+
+## Step 2: Run Analysis
+
+Let's perform our initial analysis:
+
+1. Once you have RPC server initialized, navigate to "Konveyor Analysis View" and click `Run Analysis`. Open the command palette by pressing Command + Shift + P to find it.
+   ![run_analysis](../../images/run_analysis.png)
+
+2. The Konveyor Analysis View lists issues, allowing you to filter them by file issues. On the left side, the Konveyor Issue Panel groups files based on similar issues for easier navigation.
+   ![konveyor_analysis_view](../../images/konveyor_analysis_view.png)
```

Comment:
Please check if this link is correct, I see it load the image, but I suspect you are loading a copy from: /docs/images/run_analysis.png

and NOTE /docs/scenarios/javaEE_to_quarkus/images/run_analysis.png

Reply:
This is used in other demo as well so i kept at this location. However i made copy at  `/docs/scenarios/javaEE_to_quarkus/images`